src/useLocalStorage.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+
+function getStorageValue(key, defaultValue) {
+    const saved = localStorage.getItem(key);
+    const initial = saved;
+    return initial || defaultValue;
+}
+
+export const useLocalStorage = (key, defaultValue) => {
+    const [value, setValue] = useState(() => {
+        return getStorageValue(key, defaultValue);
+    });
+
+    useEffect(() => {
+        localStorage.setItem(key, value);
+    }, [key, value]);
+
+    console.log(key);
+    return [value, setValue];
+};
```

Comment:
do we use it in the end? I remember we stopped using this solution. Pls remove if so